style-sheets/primevue-components-overrides.scss

```diff
@@ -217,7 +219,7 @@ label {
     border-radius: 0.25rem 0.25rem 0 0;
     background: map.get(lightTheme.$light-theme, 'field-01');
     width: 38rem !important;
-    height: 3rem !important;
+    // height: 2.5rem !important;
```

Comment:
The height for the dropdown in dashboard page and the dropdown in the grant access page are different. So I push a commit in FAM to specify the height for the dropdown in the dashboard page. And I feel set 2.5rem to be default seems too small for me in the grant access page, so I commented out for now.